docs/modules/releasenotes/partials/release-notes-24.2.adoc

```diff
@@ -104,3 +104,58 @@ If you want to use this feature for your Scout JS pages as well, follow these in
 Selected config properties can now be accessed from Scout JS.
 See the xref:technical-guide:common-concepts/platform.adoc#access-config-ts[Configuration documentation] on how to expose more properties to the browser and how to access them from TypeScript.
 
+== Tile Grid
+
+=== Movable (Scout JS)
+
+Tiles can now be moved to another position using drag & drop (see `TileModel.movable`).
```

Comment:
```suggestion
Tiles can now be marked as "movable", see `TileModel.movable`.
This allows them to be moved to another position using drag & drop.
```

Not all tiles are movable by default, right? It has to be enabled explicitly.

The same applies to the "resizable" chapter.